packages/cactus-plugin-ledger-connector-corda/src/test/typescript/integration/corda-v5-flow.test.ts

```diff
@@ -0,0 +1,350 @@
+import { v4 as uuidv4 } from "uuid";
+import "jest-extended";
+import {
+  CordaV5TestLedger,
+  Containers,
+  pruneDockerAllIfGithubAction,
+} from "@hyperledger/cactus-test-tooling";
+
+import {
+  IListenOptions,
+  LogLevelDesc,
+  Servers,
+} from "@hyperledger/cactus-common";
+import {
+  PluginLedgerConnectorCorda,
+  CordaVersion,
+} from "../../../main/typescript/plugin-ledger-connector-corda";
+import {
+  DefaultApi,
+  FlowStatusV1Response,
+  GetFlowV1Request,
+} from "../../../main/typescript/generated/openapi/typescript-axios/index";
+
+const logLevel: LogLevelDesc = "TRACE";
+
+import http from "http";
+import { extractShortHash } from "./../../../../../cactus-test-tooling/src/main/typescript/corda/corda-v5-test-ledger";
+import express from "express";
+import bodyParser from "body-parser";
+import { AddressInfo } from "net";
+import { Configuration } from "@hyperledger/cactus-core-api";
+
+describe("Corda Test Case", () => {
+  const cordaV5TestLedger = new CordaV5TestLedger();
+  let apiClient: DefaultApi;
+  const expressApp = express();
+  const server = http.createServer(expressApp);
+  let plugin: PluginLedgerConnectorCorda;
+
+  beforeAll(async () => {
+    const pruning = pruneDockerAllIfGithubAction({ logLevel });
+    await expect(pruning).resolves.toBeTruthy();
+    expressApp.use(bodyParser.json({ limit: "250mb" }));
+    const sshConfig = await cordaV5TestLedger.getSshConfig();
+
+    plugin = new PluginLedgerConnectorCorda({
+      instanceId: uuidv4(),
+      sshConfigAdminShell: sshConfig,
+      corDappsDir: "",
+      logLevel,
+      cordaVersion: CordaVersion.CORDA_V5,
+      apiUrl: "https://127.0.0.1:8888",
+    });
+    const listenOptions: IListenOptions = {
+      hostname: "127.0.0.1",
+      port: 0,
+      server,
+    };
+    const addressInfo = (await Servers.listen(listenOptions)) as AddressInfo;
+    const { address, port } = addressInfo;
+    const apiHost = `http://${address}:${port}`;
+    const config = new Configuration({ basePath: apiHost });
+    await plugin.registerWebServices(expressApp);
+    apiClient = new DefaultApi(config);
+    await cordaV5TestLedger.start();
+    expect(cordaV5TestLedger).toBeTruthy();
+  });
+  afterAll(async () => {
+    await cordaV5TestLedger.stop();
+    await cordaV5TestLedger.destroy();
+    await Servers.shutdown(server);
+  });
+  let shortHashID: string;
+  it("Get container", async () => {
```

Comment:
Change the description to the operation this test is meant for. For example, "Listing VNodes"